test/specs/components/Icon/Icon-test.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { isConformant, handlesAccessibility, getRenderedAttribute } from '../../commonTests'
+
+import Icon from '../../../../src/components/Icon/Icon'
+import { MenuBehavior } from 'src/lib/accessibility'
+import { getTestingRenderedComponent } from 'test/utils'
+
+describe('Icon', () => {
+  isConformant(Icon)
+
+  describe('Icon accessibility', () => {
+    handlesAccessibility(Icon, {
+      defaultRootRole: undefined,
+      accessibilityOverride: MenuBehavior,
+      overriddenRootRole: 'menu',
+    })
+
+    describe('aria hidden', () => {
+      it('icon - set to true by default', () => {
```

Comment:
better to replace 'icon' word on 'font-based' - it will be more expressive in terms of what is the difference from 'svg'